ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_manipulation.py

```diff
@@ -1009,3 +1009,35 @@ def test_fill_diagonal(
         v=v,
         wrap=wrap,
     )
+
+
+@handle_test(
+    fn_tree="trim_zeros",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=2,
+    ),
+    trim=st.sampled_from(("fb", "f", "b")),
+    ground_truth_backend="numpy",
```

Comment:
What does this do? The default `tensorflow` works fine for me, so is it necessary to use `numpy` as a ground truth backend?

Reply:
I thought to use numpy since it is the original function that has `trim_zeros` and other frameworks are implementing what numpy does.